sdk/python/feast/loaders/abstract_producer.py

```diff
@@ -47,6 +47,7 @@ def _inc_pbar(self, meta):
     def _set_error(self, exception: str):
         self.error_count += 1
```

Comment:
This line can be removed if youre throwing an exception, might be a little redundant